src/pages/home/HomePage.js

```diff
@@ -61,19 +62,33 @@ class App extends React.Component {
         this.state = {
             windowWidth: windowSize.width,
             isHamburgerEnabled: windowSize.width <= variables.mobileResponsiveWidthBreakpoint,
-            isFloatingAcionButtonActive: false,
+            isCreateMenuActive: false,
         };
 
-        this.toggleFab = this.toggleFab.bind(this);
+        this.toggleCreateMenu = this.toggleCreateMenu.bind(this);
         this.toggleHamburger = this.toggleHamburger.bind(this);
         this.dismissHamburger = this.dismissHamburger.bind(this);
         this.showHamburger = this.showHamburger.bind(this);
         this.toggleHamburgerBasedOnDimensions = this.toggleHamburgerBasedOnDimensions.bind(this);
         this.recordTimerAndToggleHamburger = this.recordTimerAndToggleHamburger.bind(this);
 
         this.animationTranslateX = new Animated.Value(
-            !props.isSidebarShown ? -300 : 0,
+            !props.isSidebarShown ? -variables.sideBarWidth : 0,
         );
+
+        // This format allows to set individual callbacks to each item
+        // while including mutual callbacks first
```

Comment:
Just so I understand... each item's callback will eventually be set here right? but because they currently have the same behaviour they are empty for now?

Reply:
I was not sure where the `menuItemData` array was meant to be declared, that's why I ended up writing it on the `HomePage` constructor. 
The [issue page](https://github.com/Expensify/Expensify.cash/issues/1240) just stated that "all menu selections should call `ChatSwitcher.show()`". I did the `Array.map()` trick so we have a template when we eventually need to call different actions for different items, but might still have a callback in common.

Reply:
This is just a temporary step. We will add two new views New Chat and New Group that will open modally in the future. But yes we should have two separate callbacks..